packages/blockchain-wallet-v4-frontend/src/scenes/TaxCenter/components/List/List.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { colors, Icon, IconName } from '@blockchain-com/constellation'
+import moment from 'moment'
+
+import { Button, Link, SpinningLoader, Text } from 'blockchain-info-components'
+
+import { Content, Description, ReportItem, StyledIcon } from './model'
+
+const COMPLETED_STATUS = 'COMPLETED'
+const PENDING_STATUS = 'PENDING'
+
+const getReportTitle = (from, to) => {
+  if (moment(to).diff(moment(from), 'year') === 0) {
+    return moment(to).format('YYYY')
+  }
```

Comment:
consider not using momentjs
https://github.com/you-dont-need/You-Dont-Need-Momentjs